MAUI/CheckBox/Visual-Customization.md

```diff
@@ -301,4 +301,26 @@ SfCheckBox checkBox = new SfCheckBox
 };
 
 {% endhighlight %}
-{% endtabs %}
\ No newline at end of file
+{% endtabs %}
+
+## Content spacing
+This feature allows users to adjust the content spacing in the [`SfCheckBox`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Buttons.SfCheckBox.html) control. 
```

Comment:
The `ContentSpacing` property is used to adjust the content spacing in the [`SfCheckBox`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Buttons.SfCheckBox.html) control.

Reply:
Updated the content.